front-end/src/components/Content/User/User.js

```diff
@@ -0,0 +1,131 @@
+import React from "react";
+import "./style.scss"
+import Header from "../../Header/Header";
+import Sidebar from "../../Sidebar/Sidebar";
+import Body from "../Body";
+import Footer from "../../Footer/Footer";
+import BreadCrumb from "../Bread-crumb/bread-crumb";
+import Pagination from "../Pagination/Pagination";
+
+export default function User () {
+  const data = [
+    {
+      checkbox: false,
+      title: "::",
+      username: "Textline",
+      emali: "Textline",
+      status: "Active",
+    },
+    {
+      checkbox: false,
+      title: "::",
+      username: "Textline",
+      emali: "Textline",
+      status: "Unactive",
+    },
+    {
+      checkbox: false,
+      title: "::",
+      username: "Textline",
+      emali: "Textline",
+      status: "Active",
+    },
+    {
+      checkbox: false,
+      title: "::",
+      username: "Textline",
+      emali: "Textline",
+      status: "Textline",
+    },
+    {
+      checkbox: false,
+      title: "::",
+      username: "Textline",
+      emali: "Textline",
+      status: "Textline",
+    },
+    {
+      checkbox: false,
+      title: "::",
+      username: "Textline",
+      emali: "Textline",
+      status: "Textline",
+    },
+    {
+      checkbox: false,
+      title: "::",
+      username: "Textline",
+      emali: "Textline",
+      status: "Textline",
+    },
+    {
+      checkbox: false,
+      title: "::",
+      username: "Textline",
+      emali: "Textline",
+      status: "Textline",
+    },
+    {
+      checkbox: false,
+      title: "::",
+      username: "Textline",
+      emali: "Textline",
+      status: "Textline",
+    },
+    {
+      checkbox: false,
+      title: "::",
+      username: "Textline",
+      emali: "Textline",
+      status: "Textline",
+    },
+  ];
+
+  return (
+    <>
+      <Header />
+      <div className='container'>
+        <Sidebar />
+        <div className="right-wrapper">
+          <BreadCrumb />
+          <div className="wrapper">
+            <table>
+              <tbody>
+                <tr>
+                  <th class="checkbox-Area">
+                    <input type="checkbox" name="scales" />
+                  </th>
+                  <th class="title_Area">Title</th>
+                  <th class="username_Area">UserName</th>
+                  <th class="email_Area">Email</th>
+                  <th class="status_Area">Status</th>
+                  <th>Actions</th>
+                </tr>
+                {data.map((item) => {
+                  return (
+                    <tr>

```

Comment:
Add note more when we loop a list we need to set a key for each item, to make sure each row is unique.
Please refer document: https://reactjs.org/docs/lists-and-keys.html#keys
```JS
 {data.map((item, idx) => {
                  return (
                    <tr key={idx}>
```
Thank you.